index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Bungee+Spice&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:wght@300;400;700;900&family=Lobster+Two:wght@400;700&family=Poppins:wght@300;400;500;600;700;800&family=Satisfy&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />  <link rel="stylesheet" href="style.css">
+  <title>Web development course</title>
+</head>
+<body>
+  <div class="head-bar">
+    <div class="head-bar-content">
+      <img src="assets/images//facebook.svg" alt="" class="head-bar-margin">
+       <p>English</p>
+       <p>My page</p>
+       <p>Log In</p>
+     </div>
+  </div>
+   <div class="wrapper">
+
+    <div class="navigation">
+      <ul class="nav-tag">
+        <li><a href="index.html" class="logo">House Of Technology</a></li>
+        <li><a href="about.html" class="active">About</a></li>
+        <li><a href="">Program</a></li>
+        <li><a href="">Join</a></li>
+        <li><a href="">Sponsor</a></li>
+        <li><a href="">News</a></li>
+        <li><button class="btn-1">Tech Campaign</button></li>
+      </ul>
+    </div>
+
+    <div id="navbar">
+      <ul class="nav-menu">
+          <li class="nav-item nav-area"><a href="about.html" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+      </ul>
+
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </div>
+      
```

Comment:
While you did a great job trying to implement this you did not follow all design guideline in you first section of the home page. Please pay attetion to each and every detail and font, fontsizes and font colors as well I advise you carefully look at the template design because your layout should be similar to design.

This is your design for the first section in you desktop version:



![home 1](https://user-images.githubusercontent.com/79506105/210609882-0cd5397e-3231-40b5-a998-67d936df2b8b.png)

Please stick to this design:
![Behance 1](https://user-images.githubusercontent.com/79506105/210610447-a7037461-19ec-4823-840b-8d81c978e718.png)